packages/app/src/specs/InlineSpecListRow.vue

```diff
@@ -33,6 +33,7 @@
           pl-8px
           hidden
           group-hover:inline
+          whitespace-nowrap
```

Comment:
I don't see a text-overflow behavior, but this can wait.

Reply:
That was to prevent this.

![Screen Shot 2021-11-09 at 5 39 16 PM](https://user-images.githubusercontent.com/25158820/141025401-2985a8bf-ae82-41a2-b8c8-a2686997366d.png)